refinery/tool_manager/selenium_tests.py

```diff
@@ -0,0 +1,186 @@
+import json
+import logging
+import time
+from urlparse import urljoin
+
+from django.conf import settings
+from django.core.management import call_command
+from django.http import HttpResponseBadRequest
+
+from django_docker_engine.docker_utils import DockerClientWrapper
+from docker.errors import NotFound
+import mock
+from rest_framework.test import force_authenticate
+
+from selenium_testing.utils import (MAX_WAIT, SeleniumTestBaseGeneric,
+                                    wait_until_class_visible)
+from tool_manager.models import Tool, ToolDefinition, VisualizationTool
+from tool_manager.tasks import django_docker_cleanup
+from tool_manager.tests import TEST_DATA_PATH, ToolManagerTestBase
+
+logger = logging.getLogger(__name__)
+
+
+class VisualizationToolLaunchTests(ToolManagerTestBase,
```

Comment:
This looks like the whole block is being moved from below? Is there anything particular I should look more closely at?

Reply:
@mccalluc Nope! just moved into its own file